products/sle/main.pm

```diff
@@ -778,6 +778,7 @@ elsif (get_var("VIRT_AUTOTEST")) {
             }
             loadtest "virt_autotest/setup_dns_service";
             loadtest "virtualization/xen/hotplugging" if get_var("ENABLE_HOTPLUGGING");
+            loadtest "virtualization/xen/storage";
```

Comment:
There should be a switch here, or once merged, it is automatically added in our official group 263,264.

Reply:
I just think it should just go to group 263.264 directly, so did not add switch. Is it really necessary to do more verification under development ?

Reply:
Seems Leon has concern for chaining it with others like hotplugging and snapshot, which is not verified in your verification jobs. Calen has concern for it too due to late public rc phase. You know it better than others. You can make the decision and take the risk. I will trust your choice.

Reply:
BTW if you want to know my suggestion, I would recommend development first also. Just to make sure everything go just green.

Reply:
Done